Gemfile.lock

```diff
@@ -66,6 +68,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
I don't think these platform-dependent extensions should be included, because it may prevent builds on other platforms, including by GitHub Pages.